ocamldoc/Makefile.docfiles

```diff
@@ -13,7 +13,7 @@
 #**************************************************************************
 
 # Define the lists of mli file used by ocamldoc to generate the stdlib
-# + otherlibs + compilerlibs + plugin hooks documentation
+# + otherlibs + compilerlibs documentation
```

Comment:
I think that the `TYPING_MLIS` and `BYTECOMP_MLIS` variables below should be removed too: they are here for the hooks in `Typemod` and `Simplif` which are now undocumented.

Reply:
Done